src/components/Sidebar/Sidebar.jsx

```diff
@@ -9,8 +9,29 @@ import { SidebarFooter } from "./Sidebar.Footer";
 import { SidebarLink } from "./Sidebar.Link";
 import { SidebarMenu } from "./Sidebar.Menu";
 import { SidebarHeading } from "./Sidebar.Heading";
+import { BellIcon } from "../../icons/BellIcon";
+import { AnnounceIcon } from "../../icons/AnnounceIcon";
+import { Drawer } from "../Drawer";
+
+export const Sidebar = ({ children, className, footer, notifications = {}, isFixed = true, onLogoClick }) => {
+    const [isNotificationsOpen, setIsNotificationsOpen] = useState(false);
+    const [isAnnouncementsOpen, setIsAnnouncementsOpen] = useState(false);
+
+    const style = useMemo(
+        () => ({
+            background: "linear-gradient(138.65deg, #583DFF 19.59%, #F849C7 62.96%, #FFC03D 97.07%)",
```

Comment:
Actually since this is a fixed value, it can be declared outside the component and it won't need memo. Just do that